master/internal/experiment_filter.go

```diff
@@ -102,7 +102,7 @@ func expColumnNameToSQL(columnName string) (string, error) {
 		"duration":        "extract(epoch FROM coalesce(e.end_time, now()) - e.start_time)",
 		"state":           "e.state",
 		"numTrials":       "(SELECT COUNT(*) FROM trials t WHERE e.id = t.experiment_id)",
-		"progress":        "COALESCE(progress, 0)",
+		"progress":        "COALESCE(progress, 0) * 100", // multiply by 100 to make it percent
```

Comment:
Filtering on a float is strange. Should this match for some request Pr all progress P where |Pr-P| < e where e is, say, 1 or 0.5? Does the web table already do this?

Reply:
I don't think web table does something like that. 
Do you think we should treat progress as integer? Then it would work as expected because progress is stored as float in db but web want it as integer.

Reply:
Yeah, I think that works.

Reply:
Prob making it integer does not work well.

Reply:
tested on local and working good with the approach